### DISCUSSION TITLE
Do I still need GitHub Education?

### DISCUSSION THREAD

**Discussion by erwanm** [community/community#24121] [GitHub Education]
<p>Hi,</p>
<p>I’m an academic researcher. I’m entitled to GitHub Pro for free thanks to GitHub <a href="https://education.github.com/benefits" rel="noopener nofollow ugc">education program</a>. I used to register as an academic in order to have private repositories, which were not available with a standard free account at the time.</p>
<p>Since private repositories are now free for everyone, I don’t think I need the GitHub Pro account anymore since i don’t use any of the other features, at least as far as I’m aware. I’m still entitled to the education program, should I renew my registration or let my account be downgraded?</p>
<p>Thanks in advance for your advice!</p>

**Comment by Zerotask** [comment]
<p>That depends on your needs.<br>
Private repos are for free now, but a Pro subscription offers more than a free tier.</p>

**Comment by nazimboudeffa** [comment]
<p>Have you listed the other benefits of beeing a Pro ?</p>

**Comment by erwanm** [comment]
<p>The benefits of GitHub Pro are <a href="https://docs.github.com/en/get-started/learning-about-github/githubs-products#github-pro" rel="noopener nofollow ugc">listed here</a>. I don’t think I need these actually.</p>

**Comment by erwanm** [comment, marked as answer]
<p>Thanks guys, I realize that my question is a bit stupid since I’m the only one who can decide what I need indeed :smiley:</p>
<p>I’m going to let the account be downgraded, I don’t need the other stuff for now.</p>
